data/open-data-resources.yaml

```diff
@@ -0,0 +1,119 @@
+- title: '1. ePSIplatform’s Cases'
```

Comment:
Is it possible and/or useful to automatically generate the number from the item's index in the list of open data resources? Dunno, might be over-engineering, but figured I'd mention it.